db/migrate/20190621140938_change_reminder_days_column_name.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeReminderDaysColumnName < ActiveRecord::Migration[5.2]
+  def up
+    rename_column :organizations, :reminder_days_before_deadline, :reminder_date
+  end
+
+  def down
+    rename_column :organizations, :reminder_date, :reminder_days_before_deadline
```

Comment:
I like the direction you're going with this -- could you use the name `:reminder_day_number`, since this isn't actually a Date field? 

Thank you! :+1: :+1:

Reply:
What about just `reminder_day` and `deadline_day`?

Reply:
That would be ok too :+1: